core/src/main/java/hudson/model/AbstractProject.java

```diff
@@ -751,8 +751,13 @@ protected void updateTransientActions() {
         for (JobProperty<? super P> p : Util.fixNull(properties))
             ta.addAll(p.getJobActions((P)this));
 
-        for (TransientProjectActionFactory tpaf : TransientProjectActionFactory.all())
-            ta.addAll(Util.fixNull(tpaf.createFor(this))); // be defensive against null
+        for (TransientProjectActionFactory tpaf : TransientProjectActionFactory.all()) {
+            try {
+                ta.addAll(Util.fixNull(tpaf.createFor(this))); // be defensive against null
+            } catch (Exception e) {
+                LOGGER.log(Level.SEVERE, "Error loading transient project action factory.");
```

Comment:
This is useless because you are not including the exception in the log message! Use the `(Level, String, Throwable)` overload.

Also it is nice to include the name of the project and the factory in the message, e.g.

    LOGGER.log(Level.WARNING, "Could not load actions from " + tpaf + " for " + this, e);